tests/installation/partitioning_raid.pm

```diff
@@ -162,31 +166,38 @@ sub run() {
         assert_screen 'partitioning-size';
         send_key 'ctrl-a';
```

Comment:
that's why I suggested `wait_screen_change`. "ctrl-a" does not care about the content but you should assert that the text got selected -> screen changed